### ISSUE TITLE
v2.0.3?

### ISSUE THREAD

**Issue by Rikk** [VREMSoftwareDevelopment/WiFiAnalyzer#219]
Why is v2.0.3 only in the Play Store? I need the normal apk (instead of "split apk" done by Google). Thanks.

[How to report a bug](https://github.com/VREMSoftwareDevelopment/WiFiAnalyzer/wiki/Feedback)

**Please search issues (open and closed) to see the issues has already been reported.**

**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Note**
Some behaviour only occurs on certain devices or versions of Android, please provide the following screenshots/information Device/Android OS (Settings → About Phone) and WiFi Analyzer version (Menu -> About)

**Additional context**

**Provide Device/Android OS information**

**Comment by HanabishiRecca** [comment]
Looks like Play version was reverted to 1.9.3 for now. Uninstall and download app again.
Or grab it [here](/VREMSoftwareDevelopment/WiFiAnalyzer/releases/tag/V1.9.3-F-DROID) if you don't want automatic update at all.

**Comment by VREMSoftwareDevelopment** [comment]
@Rikk 
Please fill the missing information in the request template we have provided above.
It is not clear to us what do mean by "normal apk"?

**Comment by Rikk** [comment]
Play Store is providing "split apk", instead of the one that contains everything and installs on all supported devices. I don't understand why you enabled it, as the size of the split apk is not much smaller.

**Comment by VREMSoftwareDevelopment** [comment]
We have asked you to fill the missing information in the request template that we have provided above.
This helps us to investigate and spend less time on the issues.

All the sources and some apks are available [here](https://github.com/VREMSoftwareDevelopment/WiFiAnalyzer/releases).
We did not enable anything special. 
[Android App Bundles](https://developer.android.com/guide/app-bundle/) maybe the reason why.
